Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -75,9 +53,83 @@ For more precisely positioning your hologram, you can use additional input from
 
 Information about where users look at is an immensely powerful tool to assess usability of a design and to identify problems in efficient work streams. 
 This tutorial discusses different eye tracking visualizations and how they fit different needs. 
-We provide basic examples for logging and loading Eye Tracking data and examples for how to visualize them. 
+We provide basic examples for logging and loading eye tracking data and examples for how to visualize them. 
+
+**Summary**: Two-dimensional attention map (heatmaps) on slates. Recording & replaying eye tracking data.
+
+<br>
+
+
+## Setting up the MRTK eye tracking samples
+
+### 1. Load EyeTrackingDemo-00-RootScene.unity
+The *EyeTrackingDemo-00-RootScene* is the base (_root_) scene that has all the core MRTK components included.
+This is the scene that you need to load first and from which you will run the eye tracking demos. 
+It comes with a graphical scene menu that allows you to easily switch between the different eye tracking samples which will be [loaded additively](https://docs.unity3d.com/ScriptReference/SceneManagement.LoadSceneMode.Additive.html).
+
+![Scene menu in eye tracking sample](../Images/EyeTracking/mrtk_et_scenemenu.jpg)
+
+The root scene includes a few core components that will persist across the additively loaded scenes, such as the MRTK configured profiles and scene camera. 
+The _MixedRealityBasicSceneSetup_ (see screenshot below) includes a script that will automatically load the referenced scene on startup. 
+By default this is _EyeTrackingDemo-02-TargetSelection_.  
+
+![Example for the OnLoadStartScene script](../Images/EyeTracking/mrtk_et_onloadstartscene.jpg)
+
+
+### 2. Adding scenes to the Build menu
+To load additive scenes during runtime, you must add these scenes to your _Build Settings -> Scenes in Build_ menu first.
+It is important that the root scene is shown as the first scene in the list:
```

Comment:
Do you know what happens if it's not (i.e. can you mention the dialog that you just added, which will appear if this is violated?).

This can help guide people on the right path (i.e. if we have a section saying here's how you know if something is going wrong, and they may search for the going wrong text, and then be led to this section about how to do things right)

Reply:
It's a good point. I leave it like this for now, mainly because this will probably change with the new changes from Lars anyways. I would like to add the revised docs for his change in a different PR though.